granule.go

```diff
@@ -76,6 +77,15 @@ func (g *Granule) Append(p *parts.Part) (uint64, error) {
 	node := g.parts.Prepend(p)
 	newSize := g.metadata.size.Add(uint64(p.Size()))
 
+	least, err := p.Least()
+	if err != nil {
+		return 0, err
+	}
+
+	if g.tableConfig.schema.RowLessThan(least, g.metadata.least) {
+		return 0, errors.New("added part to Granule that is less than least")
```

Comment:
Is there a way to pretty print the rows so we can have more information? Otherwise I think it would be interesting to also print the result of `RowLessThan(g.metadata.least, least)` to see if they are equal